manifests/config.pp

```diff
@@ -21,6 +21,16 @@
     target => $pulp::ca_cert,
   }
 
+  unless $pulp::manage_httpd {
```

Comment:
Intentional. Apparently 'manage_httpd' implies we want pulp module to configure httpd/vhost for us. Look at https://github.com/Katello/puppet-pulp/blob/master/manifests/apache.pp#L8 where it sets up the vhost. This is meant for stand alone installs, as in machines where only pulp runs.

However in the case of Katello we do not want to setup such a vhost. infact manage_httpd is set to false (https://github.com/Katello/puppet-katello/blob/master/manifests/init.pp#L114). We need pulp.conf to be setup in that scenario. I suggested creating a new var $manage_pulp_conf but @ehelms asked me to use manage_httpd instead...